src/tool/cpp/cppwinrt/main.cpp

```diff
@@ -47,8 +47,10 @@ namespace xlang
 
         settings.verbose = args.exists("verbose");
         settings.root = args.value("root", "winrt");
-        settings.input = args.files("input");
-        settings.reference = args.files("reference");
+
+        settings.input = args.files("input", database::is_database);
```

Comment:
curious why a deep inspection of the PE header is necessary, versus just an extension check?

Reply:
Previously, we just checked the extension but I was concerned about this being used with xlang (non-WinRT) projects where there has been talk of using a different file extension.